.flowconfig

```diff
@@ -14,7 +13,6 @@ module.name_mapper='\(@storybook/react-native\)' -> '<PROJECT_ROOT>/scripts/stub
 module.name_mapper='\(@skyscanner/react-native-transitiongroup\)' -> '<PROJECT_ROOT>/scripts/stubs/jsModuleStub.js'
 module.name_mapper='\(react-native-maps\)' -> '<PROJECT_ROOT>/scripts/stubs/jsModuleStub.js'
 module.name_mapper='\(recyclerlistview\)' -> '<PROJECT_ROOT>/scripts/stubs/jsModuleStub.js'
-module.name_mapper='\(bpk-react-utils\)' -> '<PROJECT_ROOT>/scripts/stubs/jsModuleStub.js'
```

Comment:
Same question here!

Reply:
We no longer depend on `bpk-react-utils` in this project.